src/components/SearchBar/SearchBar.js

```diff
@@ -10,7 +10,7 @@ import {
   StyledSearchDivider,
 } from './SearchBarStyle'
 
-const SearchBar = () => {
+const SearchBar = (props) => {
```

Comment:
All react components will take props, so although this is fine, it's not super descriptive. Instead, you can destructure what props you expect.

```
const SearchBar = ({ headerText }) => {
```

This lets us know this component expects headerText as props in the definition, instead of having to read the body to find out what props it uses.